diff --git a/nvidia-drivers-390.132-r2-OG.ebuild b/nvidia-drivers-390.132-r2.ebuild
index cdaa60bfd89..030ffcf547b 100644
--- a/nvidia-drivers-390.132-r2-OG.ebuild
+++ b/nvidia-drivers-390.132-r2.ebuild
@@ -87,7 +87,7 @@ RDEPEND="
 "
 QA_PREBUILT="opt/* usr/lib*"
 S=${WORKDIR}/
-NV_KV_MAX_PLUS="5.5"
+NV_KV_MAX_PLUS="5.5.11"
 CONFIG_CHECK="!DEBUG_MUTEXES ~!LOCKDEP ~MTRR ~SYSVIPC ~ZONE_DMA"
 
 pkg_pretend() {
